### PR TITLE
MDCT-2759 + 2760: Transition Benchmarks w/ Drawer

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Creating the `Transition Benchmarks` page with its corresponding drawer

### Related ticket(s)

MDCT-2759
MDCT-2760

---
### How to test

- Log in as a state user
- Navigate to `/wp/transition-benchmarks`

You should see the corresponding data and styling!

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by britt-mo — DISMISSED** [review]
Looked good in the browser

**Review by keeysnc — APPROVED** [review]
lgtm!

**Review by britt-mo — APPROVED** [review]

**Review by keeysnc — APPROVED** [review]
lgtm!

File: services/ui-src/src/components/menus/Sidebar.tsx [hunk 1]
```diff
@@ -70,6 +70,15 @@ export const Sidebar = ({ isHidden }: SidebarProps) => {
             section={{ name: "placeholder", path: "/standard" }}
             level={1}
           />
+          {/* Temporary Page Navigation */}
+          <NavSection
```
Comment (keeysnc) [inline services/ui-src/src/components/menus/Sidebar.tsx:74]: I think Britt has a mock route mapping for this in main.

**Review by keeysnc — APPROVED** [review]

**Review by keeysnc — APPROVED** [review]

**Pull request merged**